app/src/main/java/com/pvp/app/ui/screen/task/TaskViewModel.kt

```diff
@@ -119,7 +119,10 @@ class TaskViewModel @Inject constructor(
             taskService.merge(task)
         }
 
-        task.scheduleReminder(appContext, 10)
+        task.scheduleReminder(
+            appContext,
+            reminderTimeMinutes
+        )
```

Comment:
Well.. If you changed it here, why did you not change it in other two places? Be consistent. Line 64, 95.